apps/mobile/src/libs/styles/global-styles/flex.ts

```diff
@@ -0,0 +1,40 @@
+import type { FlexStyle } from "react-native";
```

Comment:
```suggestion
import { type FlexStyle } from "react-native";
```
We use this approach throughout the codebase